### ISSUE TITLE
Few Runtime errors

### ISSUE THREAD

**Issue by sanvenDev** [Khrylx/PyTorch-RL#10]
I received Runtime errors(invalid value in reduce).
I think it's better to use BCEwithlogits as loss criterion for discriminator, its numerically stable.

**Comment by sanvenDev** [comment]
# # #
